pkg/koordlet/statesinformer/states_device_linux.go

```diff
@@ -171,6 +191,68 @@ func (s *statesInformer) initGPU() bool {
 	return true
 }
 
+func (s *statesInformer) getGPUDriverAndModel() (string, string) {
+	if s.fakeMVML {
```

Comment:
Is this code only for testing?

Reply:
Consider adding an interface？

```
type nvmlWrapper interface {
  GetGPUDriver() string
  GetGPUModel() string
}
```

and then we can move the mock code to test file.

Reply:
fixed